platform/android/java/app/config.gradle

```diff
@@ -12,10 +12,10 @@ ext.versions = [
 
 ext.libraries = [
     androidGradlePlugin: "com.android.tools.build:gradle:$versions.androidGradlePlugin",
-    supportCoreUtils   : "com.android.support:support-core-utils:$versions.supportCoreUtils",
+    supportCoreUtils   : 'androidx.legacy:legacy-support-core-utils:1.0.0',
```

Comment:
Here and below, update the matching `$versions` constants and use them instead of the raw value.